news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups Now Open"
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 22:15:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
```

Comment:
"in mapping and modding"

may -> might

Add "Read on to find out more and how to apply!"